packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -922,132 +924,115 @@ const runGenerator = async (
-                }
+    // Copy template specific assets over.
+    const assetsDir = p.join(ASSETS_TEMPLATES_DIR, id)
+    if (sh.test('-e', assetsDir)) {
+        getFiles(assetsDir)
+            .map((file) => {
+                const relFilePath = file.replace(assetsDir, '')
+                return relFilePath
             })
-
-            // Recursively call runGenerator for the 'typescript-minimal' local dev project
-            const localDevProjectContext = {
-                ...context,
-                preset: {
-                    id: 'typescript-minimal',
-                    templateSource: {type: TEMPLATE_SOURCE_BUNDLE, id: 'typescript-minimal'},
-                    private: true
-                },
-                answers: {project: {type: 'PWAKitAppProject', name: 'local-dev-project'}}
-            }
-
-            await runGenerator(localDevProjectContext, {
-                outputDir: devOutputDir,
-                templateVersion,
-                verbose,
-                installDependencies: false
+            .forEach((relFilePath) => {
+                processTemplate(relFilePath, assetsDir, outputDir, context)
             })
+    }
 
-            // Update the typescript-minimal dev package.json with dependencies
-            updatePackageJson(p.resolve(devOutputDir, 'package.json'), {
-                devDependencies: {[answers.project.name]: 'file:../'},
-                mobify: {app: {extensions: [answers.project.name]}}
-            })
+    // Check project type and handle appropriately
+    if (answers.project.type === 'PWAKitAppExtensionProject') {
+        const devOutputDir = p.join(outputDir, LOCAL_DEV_PROJECT_DIR)
 
-            // TODO: The generator is growing, we should refactor this to be more maintainable.
-            const processGeneratedExtension = () => {
-                // do a file content replacement for extension-meta.json in the outputDir
-                // find all instances of "@salesforce/extension-base" and replace with answers.project.name
-                const extensionMetaJsonPath = p.join(outputDir, 'extension-meta.json')
-                if (fs.existsSync(extensionMetaJsonPath)) {
-                    let extensionMetaJsonContent = fs.readFileSync(extensionMetaJsonPath, 'utf8')
-                    extensionMetaJsonContent = extensionMetaJsonContent.replace(
-                        /@salesforce\/extension-base/g,
-                        answers.project.name
-                    )
-                    fs.writeFileSync(extensionMetaJsonPath, extensionMetaJsonContent)
-                }
+        // Update the root package.json to add a start script
+        updatePackageJson(p.resolve(outputDir, 'package.json'), {
+            scripts: {
+                start: `npm --prefix ./${LOCAL_DEV_PROJECT_DIR} start`,
+                'start:inspect': `npm --prefix ./${LOCAL_DEV_PROJECT_DIR} run start:inspect`
             }
+        })
```

Comment:
A lot of these changes is the result of me removing the extend conditional.. so all the code is the same pretty much, but not as indented.